Documentation/faq.md

```diff
@@ -60,8 +60,7 @@ removed from the etcd cluster member list.
 This makes sense because it's usually an application level / administrative
 action to determine whether a reconfiguration should happen based on health. 
 
-For more information, refer to
-[Documentation/runtime-reconf-design.md](https://github.com/coreos/etcd/blob/master/Documentation/runtime-reconf-design.md).
+For more information, refer to [runtime-reconf-design.md].
```

Comment:
`For more information, refer to the [runtime reconfiguration design document][runtime-reconf-design].`